infrastructure/environments/dev/.env.example

```diff
@@ -3,3 +3,6 @@ TF_VAR_CLIENT_SECRET=
 
 TF_VAR_CLUSTER_ROUTE_DESTINATION_CIDR_BLOCKS='["10.0.0.0/16"]'
 TF_VAR_CLUSTER_ROUTE_NEXT_HOP=
+
+TF_VAR_SUPPORT_EMAIL_ONE=
+TF_VAR_SUPPORT_EMAIL_TWO=
```

Comment:
It's probably a good idea to just have a single array of email addresses similar to how `TF_VAR_CLUSTER_ROUTE_DESTINATION_CIDR_BLOCKS` works above

Reply:
it need to produce multiple blocks in the terraform config, will that be easy ?

Reply:
yes please to pairing @sgrowe

Reply:
It's shouldn't too hard, I think this something like this code should do the trick:

```tf
resource "azurerm_monitor_action_group" "support" {
  name                = "CriticalAlertsAction"
  resource_group_name = var.resource_group_name
  short_name          = "support"

  dynamic "email_receiver" {
    for_each = var.support_emails

    content {
      name                    = email_receiver.value
      email_address           = email_receiver.value
      use_common_alert_schema = true
    }
  }
}
```